### ISSUE TITLE
Command to jump to current file in browser

### ISSUE THREAD

**Issue by apolzon** [aziz/SublimeFileBrowser#57]
Is there a way to map a key binding to open the file browser to the currently active file? ala the "Reveal in sidebar" option in the sublime context menu for a given file.

Otherwise, what an awesome tool.

**Comment by apolzon** [comment]
Thanks! It may be worth noting that this does not work if the "project" setting is set to true.

**Comment by vovkkk** [comment]
@apolzon do you mean when project contains several folders and you choose one from quick panel? because if there is just one folder it works for me.

**Comment by apolzon** [comment]
My project is a rails application -- it has multiple top-level folders, each with a deep hierarchy. If I am in a deep file (ie spec/controllers/blah.rb), and open file browser with the project setting enabled, it opens the root of my project. Once I disabled the project setting, it worked as expected.

I can't really remember why I enabled the project setting, so this is not a big deal to me at the moment, just thought I'd mention to find out if it was intended behavior or a bug.
